pyaedt/icepak.py

```diff
@@ -124,7 +128,16 @@ def assign_openings(self, air_faces):
         Returns
         -------
         type
```

Comment:
As you're adding these examples, if you see incorrect or generic returns, please modify them to the correct return type using the sphinx `:class:` directive.

```suggestion
        :class:`pyaedt.modules.Boundary.BoundaryObject`
```

Reply:
@akaszynski I will do that moving forward. I didn't know about referencing classes like that.